test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -59,9 +59,12 @@ fromSanitizedHeaders<TestRequestHeaderMapImpl>(const test::fuzz::Headers& header
 Http1Settings fromHttp1Settings(const test::common::http::Http1ServerSettings& settings) {
   Http1Settings h1_settings;
 
+  // When in future allowing trailers, then the kTrailers case in directionalAction() needs change,
+  // too.
```

Comment:
This PR enables trailers for H1 if they're in the settings proto, so this comment seems incorrect.

Reply:
Correct, deleted.